webserver/external/messybrainz.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import ujson
+
+def get_messybrainz_id(querydata):
+    """
+    This function queries messybrainz to get a msid according to the 'artist/title' present in the item metadata
+    Args:
+        querydata: A dictionary containing at least artis and song title
+
+    Returns:
+        msid: string messybrainz id -> if it is found in the messybrainz database
+        None: if no reference is fount in messybrainz database
+
+    """
+    msurl = 'https://messybrainz.org/submit'
+    if isinstance(querydata, list):
+        if 'artist' in querydata[0].keys() and 'title' in querydata[0].keys():
+            jsondata = ujson.dumps(querydata)
+            r = requests.post(msurl, data=jsondata)
+            if r.status_code == 200:
+                #get the correct id
+                responsedata = r.json()
+                if 'recording_mbid' in responsedata['payload'][0]['ids'].keys() and 'recording_msid' in responsedata['payload'][0]['ids'].keys():
+                    if responsedata['payload'][0]['ids']['recording_mbid'] != '':
+                        msid = responsedata['payload'][0]['ids']['recording_mbid']
```

Comment:
I put this as TODO once we merge the PR195 that already implement the changes to accept the type of submission in the [submit_low_level_data](https://github.com/metabrainz/acousticbrainz-server/pull/195/files#diff-973d6aba9f5f7b7acda5e7780e0c45a7R129)